src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -170,6 +172,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
     [OuterLoop]
     public static void SendTimeout_Zero_Throws_TimeoutException_Immediately()
     {
+        TimeSpan serviceOperationTimeout = TimeSpan.FromMilliseconds(10000);
         BasicHttpBinding binding = new BasicHttpBinding();
```

Comment:
Sorry, I was talking about the test that sets a zero timeout and expects immediate timeout.  I think when everything works properly, the service operation will probably not even get called.  But if it did, I think it would stall the full 10 seconds before noticing the client connection was lost.  I just wanted to avoid sending the service operation to sleep for an unnecessarily long time.  Not a blocking issue for me, just a suggestion.